### DISCUSSION TITLE
请教，如何获得一个自动占满窗口大小的div ?

### DISCUSSION THREAD

**Discussion by HuangLei0x2e5** [layui/layui#2142] [讨论]
我的问题是这样：因为我的内容是动态加载的，所以我希望有一个自动占满窗口空白大小的div，如果内容太多，就显示滚动条
现在的问题是，我引用layui.css之后，body的高度会自动收缩到最小

我应该怎么设置呢

**Comment by HuangLei0x2e5** [comment]
我找到一个办法

      let viewHeight = layui.$(window).height();
      layui.$('#myCard').height(viewHeight - 160);
      
    我感觉不太好，但是起码有效
    
   大神们有没有更优雅的办法？

**Comment by Sight-wcg** [comment]
https://developer.mozilla.org/zh-CN/docs/Web/CSS/calc
